modules/standard/Math.chpl

```diff
@@ -769,6 +769,32 @@ module Math {
   }
 
 
+  /* e - exp(1) or  the base of the natural logarithm */
+  param e = 2.7182818284590452354;
+  /* log2(e) */
+  param log2e = 1.4426950408889634074;
+  /* log10(e) */
+  param log10e = 0.43429448190325182765;
+  /* log(2) (natural logarithm) */
+  param ln_2 = 0.69314718055994530942;
+  /* log(10) (natural logarithm) */
+  param ln_10 = 2.30258509299404568402;
+  /* pi - the circumference/the diameter of a circle */
+  param pi = 3.14159265358979323846;
+  /* pi/2 */
+  param pi2 = 1.57079632679489661923;
+  /* pi/4 */
+  param pi4 = 0.78539816339744830962;
```

Comment:
Yes, I was trying to follow the C names since people might be familiar with those... but that doesn't always work, and these aren't exactly the same already (in C it's M_PI).